src/Attachment.php

```diff
@@ -7,7 +7,7 @@
 use Psr\Http\Message\StreamInterface;
 
 
-Final class Attachment extends AbstractNode {
+final class Attachment extends AbstractNode {
```

Comment:
specificity over caps... This really should come with a coding style issue and PR first

Reply:
According to the PHP sample code in official PHP [website](http://php.net/manual/en/language.oop5.final.php), the keyword is ```final``` .

Reply:
it was running without error peter, again it was needless specificity it fixed nothing